packages/electron-updater/src/AppImageUpdater.ts

```diff
@@ -93,7 +93,7 @@ export class AppImageUpdater extends BaseUpdater {
       destination = path.join(path.dirname(appImageFile), path.basename(options.installerPath))
     }
 
-    execFileSync("mv", ["-f", options.installerPath, destination])
+    renameSync(options.installerPath, destination)
```

Comment:
This doesn't seem like it would work due to `installerPath` being a space-escaped string for piping to the cmd shell
> If the platform is Linux, replace spaces with '\ ' for shell compatibility

https://github.com/electron-userland/electron-builder/blob/6a6bed46c428b45105ada071a9cb89b5d4f93d9e/packages/electron-updater/src/BaseUpdater.ts#L52-L60

Reply:
Yes, I found that this was the cause, so I changed it to draft.

I don't think it's handled well here - it shouldn't directly add /. Instead, it should be handled by each deb/rpm/pacman separately. 

These linux install commands use string concatenation, so if malicious code is added to the name, it could lead to serious security vulnerabilities.

https://github.com/electron-userland/electron-builder/blob/6a6bed46c428b45105ada071a9cb89b5d4f93d9e/packages/electron-updater/src/PacmanUpdater.ts#L34-L35